anvil/catalog/ftd_consensus/models/GREGoR_R03_GRU_20250612/GREGoR_R03_GRU_20250612_ftd_accesspolicy.sql

```diff
@@ -1,22 +1,22 @@
 {{ config(materialized='table', schema='GREGoR_R03_GRU_20250612_data') }}
 
     with source as (
-        select 
+        select DISTINCT
     --     GEN_UNKNOWN.disease_limitation::text as "disease_limitation",
-    CASE participant.consent_code
-        WHEN 'GRU' THEN 'General Research Use'
-        WHEN 'HMB' THEN 'Health/Medical/Biomedical'
-        WHEN 'DS' THEN 'Disease-Specific (Disease/Trait/Exposure)'
-        WHEN 'IRB' THEN 'IRB Approval Required'
-        WHEN 'PUB' THEN 'Publication Required'
-        WHEN 'COL' THEN 'Collaboration Required'
-        WHEN 'NPU' THEN 'Not-for-profit use only'
-        WHEN 'MDS' THEN 'Methods'
-        WHEN 'GSO' THEN 'Genetic Studies only'
-        WHEN 'GSR' THEN 'Genomic Summary Results'
-       END::text as "description",
-    --    GEN_UNKNOWN.website::text as "website",
-      {{ generate_global_id(prefix='ap',descriptor=['participant.consent_code'], study_id='GREGoR_R03_GRU_20250612') }}::text as "id"
+        CASE participant.consent_code
+            WHEN 'GRU' THEN 'General Research Use'
+            WHEN 'gru' THEN 'Health/Medical/Biomedical'
+            WHEN 'DS' THEN 'Disease-Specific (Disease/Trait/Exposure)'
+            WHEN 'IRB' THEN 'IRB Approval Required'
+            WHEN 'PUB' THEN 'Publication Required'
+            WHEN 'COL' THEN 'Collaboration Required'
+            WHEN 'NPU' THEN 'Not-for-profit use only'
+            WHEN 'MDS' THEN 'Methods'
+            WHEN 'GSO' THEN 'Genetic Studies only'
+            WHEN 'GSR' THEN 'Genomic Summary Results'
+        END::text as "description",    
+        --    GEN_UNKNOWN.website::text as "website",
```

Comment:
You might ask in a PL if it is okay for tgt model columns to be missing. To my knowledge all of the fields should exist in the harmonized model, but if there is no known data for them, the field should then be set to NULL.